apps/backend/src/config.ts

```diff
@@ -97,20 +97,29 @@ export const config = {
       ),
     },
     forbiddenExtensions: env('FORBIDDEN_EXTENSIONS', '').split(','),
+    taskManagerMaxRetries: Number(env('TASK_MANAGER_MAX_RETRIES', '3')),
   },
-  services: {
-    taskManager: {
-      active:
-        (optionalBoolEnvironmentVariable('TASK_MANAGER_ACTIVE') ||
-          optionalBoolEnvironmentVariable('ALL_SERVICES_ACTIVE')) &&
-        !optionalBoolEnvironmentVariable('TASK_MANAGER_DISABLED'),
-      maxRetries: Number(env('TASK_MANAGER_MAX_RETRIES', '3')),
-    },
-    objectMappingArchiver: {
-      active:
-        (optionalBoolEnvironmentVariable('OBJECT_MAPPING_ARCHIVER_ACTIVE') ||
-          optionalBoolEnvironmentVariable('ALL_SERVICES_ACTIVE')) &&
-        !optionalBoolEnvironmentVariable('OBJECT_MAPPING_ARCHIVER_DISABLED'),
+  featureFlags: {
+    flags: {
+      taskManager: {
+        active:
+          (optionalBoolEnvironmentVariable('TASK_MANAGER_ACTIVE') ||
+            optionalBoolEnvironmentVariable('ALL_SERVICES_ACTIVE')) &&
+          !optionalBoolEnvironmentVariable('TASK_MANAGER_DISABLED'),
+      },
+      objectMappingArchiver: {
+        active:
+          (optionalBoolEnvironmentVariable('OBJECT_MAPPING_ARCHIVER_ACTIVE') ||
+            optionalBoolEnvironmentVariable('ALL_SERVICES_ACTIVE')) &&
+          !optionalBoolEnvironmentVariable('OBJECT_MAPPING_ARCHIVER_DISABLED'),
+      },
+      buyCredits: {
+        active: optionalBoolEnvironmentVariable('BUY_CREDITS_ACTIVE'),
+        employeeOnly: optionalBoolEnvironmentVariable(
+          'BUY_CREDITS_EMPLOYEE_ONLY',
+        ),
+      },
     },
+    employeeDomains: env('BUY_CREDITS_EMPLOYEE_DOMAINS', '').split(','),
```

Comment:
Feature design:
Should the allow list also allow specific email addresses?
For example, if we had volunteers/ambassadors/etc who we wanted to give access to test the system.

Optional nitpick:
Would it be shorter and easier to spell if we called this "staff" instead?